app/views/library_hours/show.html.erb

```diff
@@ -1,57 +1,96 @@
-<div class="row">
-  <div class="col breadcrumbs">
-    <%= link_to "Home", root_path.to_s %> 
-    <%= link_to "Hours", controller: 'library_hours', action: :index %> 
-    <%= @location.first.name %>
-  </div>
-</div>
-<div class="row">
-  <div class="col">
-    <div class="vertical-stripes-sm">
-      <h2><span><%= @location.first.name %> Hours</span></h2>
+<%# binding.pry %>
```

Comment:
Let's take commented out break points out of PR'ed code.

Reply:
And please resolve conflicts.

Reply:
I don't have that binding.pry in my code to remove... Is that an outdated file you were looking at?

Reply:
It's gone now. Hmmm.